R/bundle.R

```diff
@@ -465,7 +465,7 @@ inferDependencies <- function(appMode, hasParameters, python) {
 inferPythonEnv <- function(workdir, python) {
   # run the python introspection script
   env_py <- system.file("resources/environment.py", package = "rsconnect")
-  args <- c(env_py, workdir)
+  args <- c(shQuote(env_py), shQuote(workdir))
```

Comment:
FYI: `shQuote()` is vectorized so you can just call `shQuote(c(env_py, workdir))`. (The code as written is still correct so no need to change; just want to make sure you're aware of some of the R idioms)